enumerables.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < size
+      yield(self[i])
+      i += 1 
+    end
+    self
+  end
+
+  def my_each_whit_index
+    i = 0
+    while i < size 
+      yield(self[i], i)
+      i += 1
+    self
+    end
+  end
+
+  def my_select 
+    result = []
+    my_each {|item| return.push(item) if yield (item) }
```

Comment:
my_select method should also return an enumerable if no block is given.
check your code on line 24, it should be **```result.push```**